index.html

```diff
@@ -492,6 +498,11 @@ <h1 class="page-header">Red October Management</h1>
 					data.Labels[i] = data.Labels[i].trim();
 					if (data.Labels[i] == "") { data.Labels.splice(i, 1); }
 				}
+				data.Usages = data.Usages.split(',');
+				for(var i=0, l=data.Usages.length; i<l; i++){
+					data.Usages[i] = data.Usages[i].trim();
+					if (data.Usages[i] == "") { data.Usages.splice(i, 1); }
```

Comment:
`data.Usages[i] == ""` should be `data.Usages[i] === ""`. JavaScript is weird with equality.

    0 == "" // Returns true
    [] == "" // Returns true